src/codegen/forward.jl

```diff
@@ -63,16 +64,13 @@ function fwd_transform!(ci, mi, nargs, N)
     end
 
     meth = mi.def::Method
-    nargs = Int(meth.nargs)
-    for i = 1:nargs
-        if meth.isva && i == nargs
-            args = map(i:(nargs+1)) do j::Int
```

Comment:
This is wrong cos `nargs` is about the method. It doesn't know about how many varargs in this call